pkg/services/publicdashboards/models/models_test.go

```diff
@@ -15,36 +17,88 @@ func TestPublicDashboardTableName(t *testing.T) {
 
 func TestBuildTimeSettings(t *testing.T) {
 	var dashboardData = simplejson.NewFromAny(map[string]interface{}{"time": map[string]interface{}{"from": "2022-09-01T00:00:00.000Z", "to": "2022-09-01T12:00:00.000Z"}})
-	fromMs, toMs := internal.GetTimeRangeFromDashboard(t, dashboardData)
+	defaultFromMs, defaultToMs := internal.GetTimeRangeFromDashboard(t, dashboardData)
+
+	selectionFromMs := strconv.FormatInt(time.Now().UnixMilli(), 10)
+	selectionToMs := strconv.FormatInt(time.Now().Add(time.Hour).UnixMilli(), 10)
+
 	testCases := []struct {
 		name       string
 		dashboard  *models.Dashboard
 		pubdash    *PublicDashboard
 		timeResult TimeSettings
+		reqDTO     PublicDashboardQueryDTO
 	}{
 		{
 			name:      "should use dashboard time if pubdash time empty",
 			dashboard: &models.Dashboard{Data: dashboardData},
-			pubdash:   &PublicDashboard{},
+			pubdash:   &PublicDashboard{TimeSelectionEnabled: false},
 			timeResult: TimeSettings{
-				From: fromMs,
-				To:   toMs,
+				From: defaultFromMs,
+				To:   defaultToMs,
 			},
+			reqDTO: PublicDashboardQueryDTO{},
 		},
 		{
 			name:      "should use dashboard time even if pubdash time exists",
```

Comment:
could we update the test name? Maybe something like `should use dashboard time when time selection is disabled`

Reply:
or just a blurb mentioning that its when time selection is disabled